CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.2)
 
+enable_language(CXX)
```

Comment:
Otherwise Conan doesn't work.

Reply:
The CMake docs say [this is the same](https://cmake.org/cmake/help/latest/command/enable_language.html) as using the `project()` command, which we already do here:
https://github.com/Project-OSRM/osrm-backend/blob/06b1b980bbdfc838c4f8e8cb370cba6dfb0329b0/CMakeLists.txt#L78

Any idea why Conan specifically needs this?